src/mtype.d

```diff
@@ -7979,8 +7979,9 @@ public:
             e = e.semantic(sc);
             return e;
         }
-        auto de = new DotVarExp(e.loc, e, d);
```

Comment:
`de` is DotVarExp, so we cannot assign the `de.semantic` result to it. So, if we want to print the `de.semantic` result, the original code should be rewirten to:

```d
        auto de = new DotVarExp(e.loc, e, d, d.hasOverloads());
-       return de.semantic(sc);
+      e = de.semantic(sc);
+      printf("%s", e.toChars());
+      return e;
```

By reusing `Expression e` variable, now we can insert one `printf` line.

```d
        e = new DotVarExp(e.loc, e, d, d.isFuncDeclaration() !is null);
        e = e.semantic(sc);
+      printf("%s", e.toChars());
        return e;
```